src/sbmlutils/resources/templates/macros_sbml.html

```diff
@@ -245,6 +246,24 @@
 
 	<td>{{ item.sbo }}</td>
 	<td>{{ item.cvterm }}</td>
+    <td>
```

Comment:
This is again too much code duplication!
Just define once a macro for the uncertainties and then reuse it on all objects!
See the `{{ id_info(item) }}` macro for an example.
It is defined once at the top and then reused in the other macros. Do the same strategy here instead of copying the same block everywhere.